README.md

```diff
@@ -25,3 +25,60 @@ Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protrac
+
+<hr style="height:0.3px;border:none;color:lightgrey;background-color:lightgrey;" />
+
+<p align="center">
+<a href="mailto:sales@syncfusion.com?Subject=Syncfusion Angular Card - GitHub" target="_top">sales@syncfusion.com</a> | <a href="https://www.syncfusion.com?utm_source=github&utm_medium=listing&utm_campaign=angular-card-github-samples">www.syncfusion.com</a> | Toll Free: 1-888-9 DOTNET <br>
```

Comment:
Check on all UTM Parameter links, and use the bellow.
![image](https://github.com/SyncfusionExamples/ej2-angular-11-card/assets/95272441/5940bd27-1676-4258-b350-12ad294f462d)

Reply:
changed